test/extensions/filters/http/custom_response/BUILD

```diff
@@ -76,6 +76,7 @@ envoy_extension_cc_test(
 
 envoy_extension_cc_test(
     name = "custom_response_integration_test",
+    size = "large",
```

Comment:
this one just flaked here https://dev.azure.com/cncf/envoy/_build/results?buildId=131505&view=logs&j=d1f76054-8f79-554b-6f4a-11d6a63b8e00&t=f1bd857c-4611-5620-0b5a-d15324470098&l=1517

so apart from that we probably need to revert it the download minimal PR doesnt seem to have helped